.gitignore

```diff
@@ -17,6 +17,7 @@ jspm_packages
 .node_repl_history
 
 # SKY UX temp directory
+.skypagesdocs
```

Comment:
I can try it. I was worried that another process would prematurely delete the `.skyuxtmp` directory.

Reply:
I just tested this with `.skypagestmp/docs/` and it works!

Reply:
Actually, this does not work for `skyux build-public-library` because that command clears out the `.skypagestmp` directory and the build fails. I think we'll need to create a custom folder for the documentation generation just to make sure the processes don't "cross streams".